ocaml/sdk-gen/go/test_gen_go.ml

```diff
@@ -934,6 +940,158 @@ module StringOfTyWithEnumsTest = struct
     ]
 end
 
+module GroupParamsTest = Generic.MakeStateless (struct
```

Comment:
It would be nice to add some cases, like zero parameters, 1 parameter, numbered, rel_code + numbered, etc.

Reply:
Added